civicrm_resources/civicrm_resources.module

```diff
@@ -23,24 +23,30 @@ function civicrm_resources_add_resources($extention_path, $req_files = []) {
   $js_list = [];
   $css_list = [];
 
+  $resourceFilenameUri = array_map(function ($file) {
+    return ['filename' => $file->filename, 'uri' => $file->uri];
+  }, $files);
+  $resourceFilenameUriValues = array_values($resourceFilenameUri);
+  $fileNames = array_column($resourceFilenameUri, 'filename');
```

Comment:
We'll have a problem here if have two files with the same name, but in different folders

Reply:
Yes that is true but since `array_search` is used to [find index](https://github.com/compucorp/civihr-employee-portal/pull/535/files#diff-ec67a98d0a58dd334cfd09811516a417R32) of the file, it will always return the first corresponding key if successful. So the second will always be ignored.